### PR TITLE
📝 Document org transfer

### PR DESCRIPTION
**Pull request by CollierCZ**
## Why

It wasn't clear how to transfer ownership of an organization. See context.

## What's changed

Added a section on how to transfer. Now that it's getting bigger and bigger, we should consider breaking the page up into smaller pages for each task. But that's outside the scope of this small change.

### CODE REVIEW

**Review by trolologuy — CHANGES_REQUESTED** [review]
The original split between users and orgs was a bit unclear, the current change to put everything org-related under the organization page would definitely help.

- [x] Final built pages are working as expected.
- [x] The information is correct.
- [] The information is clear.

**Comment by CollierCZ** [comment]
I feel like the information that wasn't clear was kind of out of the scope of this issue. I changed it quickly, so should be better now. But I feel like the checkbox that shouldn't have been checked was this one:

- [ ] All links work, including to section headers.

That's cause I was relying too heavily on the link checker, rather than realizing it was in a codetab :facepalm: Good catch.

File: docs/src/administration/organizations.md [hunk 1]
```diff
@@ -219,7 +219,17 @@ platform organization:delete --org acme
...
+2. Submit a [support ticket](https://console.platform.sh/-/users/~/tickets) from your organization account to ask for the transfer.
+
+Once the transfer is completed, the new organization can administer all project settings and billing and receives future invoices.
 Ownership transfer automatically transfers subscription charges to the new organization.
+
+## Transfer organization ownership
+
+To transfer an organization to a different owner, first make sure that user is part of the organization.
+If they aren't yet, [add them](#manage-your-organization-users).
+Then submit a [support ticket]((https://console.platform.sh/-/users/~/tickets) from the current organization to ask for the transfer.
```
Comment (trolologuy) [inline docs/src/administration/organizations.md:235]: :clipboard:
```suggestion
Then submit a [support ticket](https://console.platform.sh/-/users/~/tickets) from the current organization to ask for the transfer.
```

File: .platform/routes.yaml [hunk 2]
```diff
@@ -115,6 +115,9 @@ https://{default}/:
       # Move CLI from Development to Administration in August 2022
       "/development/cli(.*)":  { "to": "/administration/cli$1", "code": 301, "regexp": true }
 
+      # Move project ownership transfer to orgs in August 2022
+      "/administration/users.html#transfer-project-ownership":  { "to": "/administration/organizations.html#transfer-project-ownership", "code": 301 }
```
Comment (trolologuy) [inline .platform/routes.yaml:119]: :lady_beetle:
The 301 doesn't seem to work on my end 🤔 
Opening:
https://move-that-org-pd7yzoa-652soceglkw4u.eu-3.platformsh.site/administration/users.html#transfer-project-ownership

does nothing

**Review by trolologuy — CHANGES_REQUESTED** [review]
Two smaller issues left, other than that ready to 🚢 ✅ 

- [x] Final built pages are working as expected.
- [x] The information is correct.
- [x] The information is comprehensive.
- [x] The information is concise.
- [x] The information is clear.
- [ ] If anything has been moved or removed, there are clear redirects to the new location or related content.
- [x] All links work, including to section headers.
- [x] The changes follow the content and markup style guides.
- [x] Any relevant issues raised by the linting have been addressed.
- [x] The page fits the content model defined through the templates.

File: .platform/routes.yaml (hunk 2 above)
Comment (CollierCZ) [reply .platform/routes.yaml:119]: That's why I'd removed the request for review before you posted this :laughing:

Comment (CollierCZ) [reply .platform/routes.yaml:119]: Now I think I hadn't pushed correctly :thinking:

**Review by trolologuy — APPROVED** [review]
✅ 
- [x] Final built pages are working as expected.
- [x] The information is correct.
- [x] The information is comprehensive.
- [x] The information is concise.
- [x] The information is clear.

**Pull request merged**